doc/guidelines_services.md

```diff
@@ -20,6 +23,6 @@
   - Gitea für interne und private Repos.
 
 - NextCloud:
-  - Jede aktive Person kann einen Account für FOSS-AG Dinge bekommen
-  - Datenschutz muss gewahrt werden
+  - Jede Person kann auf Anfrage einen Account für FOSS-AG Dinge bekommen mit Vetorecht unter Stimmberechtigten.
+  - Datenschutz muss gewahrt werden.
```

Comment:
Die einzige Stelle, wo ich etwas in Nachdenken komme.
Wenn wir das so übernehmen, würde ich gerne die Gruppen Aufteilung der Nextcloud sowie die Ordnerberechtigungen durchgehen.

Reply:
Sonst finde ich diese Variante in Ordnung